### PR TITLE
Dynaconf conversion for upgrade section

### PR DESCRIPTION
**Pull request by devendra104**

### CODE REVIEW

File: conf/upgrade.yaml.template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+UPGRADE:
+  # Base version of the Satellite, Capsule.
+  # FROM_VERSION: "6.8"
+  # Target version of the Satellite, Capsule.
+  # TO_VERSION: "6.9"
+  # Satellite, Capsule hosts RHEL operating system version.
+  # OS: "rhel7"
+  # Capsule's activation key, These keys will only available if we create the satellite vm -
```
Comment (ColeHiggins2) [inline conf/upgrade.yaml.template:8]: Think you have a typo. Is it supposed to say "These keys will only "be" available?

**Review by JacobCallahan — APPROVED** [review]

**Review by ColeHiggins2 — APPROVED** [review]
Looks good to me!

**Comment by JacobCallahan** [comment]
@devendra104 is this still waiting on something?

**Comment by devendra104** [comment]
> @devendra104 is this still waiting on something?

Now we are good to merge.

**Pull request merged**